docs/cookbook.rst

```diff
@@ -249,3 +249,33 @@ Unless of course something bad happens inside of ``PerformA`` in which case the
     AssertionError:
     Expected: <True>
          but: was <False>
+
+
+Using Either
+===========
+
+Sometimes we need to try one thing then followed by a different thing when the first one fails.
+You could just add a try/except in your code but that breaks up the screenplay pattern.
+So there is an action for that; :func:`~screenpy.actions.Either`::
```

Comment:
That starts to imply we know WHY the tester is using try/except; we can't know all the reasons.  to stay agnostic for fear of treading into judgement.  It is completely possible try/except is needed for something other than state management.  Maybe it simply needs to say something like:

> Sometimes you need to use `try/except` logic in your code but doing so disrupts the screenplay pattern readability.  Using the action :func:`~screenpy.actions.Either`: should suffice in most cases.

Reply:
How about a compromise?
> Sometimes you may be tempted to reach for an `if`/`else` or a `try`/`except` control flow in your test, for one reason or another. Luckily, your Actor can perform this flow with the :class:`~screenpy.actions.Either` Action!

Reply:
I guess that's the same information as what you had, just with a different voice. Your rewrite reads clearly, we can use that one!

Reply:
Take a look at the latest update.  I'm leaving out the if/else for now to avoid confusion.  I'm still convinced there is a way to create an action to cover those.